src/providers/on-chain-quote-provider.ts

```diff
@@ -611,6 +633,12 @@ export class OnChainQuoteProvider implements IOnChainQuoteProvider {
             (route as MixedRoute).pools.some((pool) => pool instanceof V4Pool)
         )
       : false;
+    const protocol = useMixedRouteQuoter
+      ? Protocol.MIXED
```

Comment:
this is so hard to read, can we just do if else? 😅

Reply:
i think i was using ternary instead of if-else for conciseness.. because otherwise, it would look like the following:

```
let protocol = undefined;

    if (useMixedRouteQuoter) {
      protocol = Protocol.MIXED;
    } else {
      protocol = useV4RouteQuoter ? Protocol.V4 : Protocol
    }
```

i wanted to avoid protocol being a mutable variable.